content/en/docs/reference/access-authn-authz/authorization.md

```diff
@@ -17,7 +17,7 @@ policies using the supported authorization modules.
 {{% capture body %}}
 In Kubernetes, you must be authenticated (logged in) before your request can be
 authorized (granted permission to access). For information about authentication,
-see [Accessing Control Overview](/docs/reference/access-authn-authz/controlling-access/).
+see [Access Control Overview](/docs/reference/access-authn-authz/controlling-access/).
```

Comment:
I looked at the title of the page this links to.
```suggestion
see [Controlling Access to the Kubernetes API](/docs/reference/access-authn-authz/controlling-access/).
```
?

Reply:
Good suggestion. Done.